integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/strategy/DpvStatusStrategy.java

```diff
@@ -25,6 +27,7 @@
 @Component
 public class DpvStatusStrategy implements StatusStrategy {
     private static final ServiceIdentifier serviceIdentifier = ServiceIdentifier.DPV;
+    private static final Logger log = LoggerFactory.getLogger(DpvStatusStrategy.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "log" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)